src/games/strategy/triplea/ai/fastAI/FastAI.java

```diff
@@ -9,14 +9,16 @@
  */
 public class FastAI extends ProAI {
 
+  // Odds estimator
+  private final static IOddsCalculator estimator = new FastOddsEstimator();
+
   public FastAI(final String name, final String type) {
     super(name, type);
   }
 
   @Override
-  public void initializeBattleCalculator() {
-    final IOddsCalculator calc = new FastOddsEstimator();
-    ProBattleUtils.setOddsCalculator(calc);
+  protected void initializeCalc() {
+    calc = new ProOddsCalculator(estimator);
```

Comment:
So really what I should eventually do is rename 'ProAI' to something like 'CoreAI' then have both ProAI and FastAI extend it and implement certain methods. The relationship between them is pretty much inheritance as they really share the same methods and flow with just a few differences.

At this point, for the AI inheritance makes more sense than IOC as they share 99% of the same code so would tie all the same modules except for one in. IOC would end up with more code duplication since all the glue logic except that 1% would be the same.